src/app/routes/utils/regex/utils/index.js

```diff
@@ -38,24 +38,28 @@ const homePageServices = [
   'afaanoromoo',
   'afrique',
   'amharic',
+  'arabic',
```

Comment:
Should this PR include these other services?

Reply:
Ah, so this branch is branching off a PR (due for release on 29 Feb) and that PR contains, arabic, hundi and mundo

Reply:
Could we change the base branch to WSTEAM-1051-launch-mundo-arabic-hindi-homepages, so that those changes won't appear? (Alternatively we wait until after Thurs, when those changes will be on latest anyway...?)

Reply:
Base branch changed!

Reply:
Apologies I may be misunderstanding, but with the base branch set to `WSTEAM-1051-launch-mundo-arabic-hindi-homepages`, when that gets merged won't that branch get automatically deleted? This will point to the deleted branch, so when this tries to get merged what will happen?

Reply:
When the base branch is merged and is deleted, GitHub automatically resets the base branch to latest.

Reply:
Ah cool, didn't know that

Reply:
Here's an example: https://github.com/bbc/simorgh/pull/11277

This will have coincided with the original base branch being merged.

![image](https://github.com/bbc/simorgh/assets/58214768/45850d88-3450-4592-9209-7f99658a679d)